shared/actions/chat2/index.tsx

```diff
@@ -28,6 +28,7 @@ import * as TeamsConstants from '../../constants/teams'
 import logger from '../../logger'
 import {isMobile, isIOS} from '../../constants/platform'
 import {NotifyPopup} from '../../native/notifications'
+import {Buffer} from 'buffer/'
```

Comment:
note this is tricky. we need the `/` at the end so it doesn't pick it up as the node module and instead uses our polyfill that we need for mobile